app/javascript/stores/components/new-product-form.vue

```diff
@@ -181,20 +206,26 @@ export default {
       name: this.product.name,
       link: this.product.link,
       price: this.product.price,
+      category_id: this.product.category_id,
     } : {
       csrfToken: getCsrfToken(),
       imageData: '',
       errors: {},
       name: '',
       link: '',
       price: null,
+      category_id: null,
     };
   },
   props: {
     product: {
       type: Object,
       default: null,
     },
+    categories: {
+      type: Array,
+      default: [],
```

Comment:
Type of the default value for 'categories' prop must be a function  vue/require-valid-default-prop